cms/templates/js/certificate-editor.underscore

```diff
@@ -0,0 +1,45 @@
+<form class="collection-edit-form certificate-edit-form">
+    <% if (error && error.message) { %>
+        <div class="certificate-edit-error message message-status message-status error is-shown" name="certificate-edit-error">
```

Comment:
A screen reader user will not be aware of any message that appears here since their screen reader will only read what is currently in focus.   ```aria-live``` attribute to the rescue, it tells the screen reader to read text that gets injected into the DOM inside the element with the attribute.  Problem with ```aria-live``` is that it cannot be used on the node that is being injected, it has to be used on a node that already exists in the DOM.  Can you create a parent ```<div aria-live="polite">``` here, outside the conditional?  Anything that gets added to that div, like this error message, will be read to SR user as soon as the SR stops talking.